release/mk-release-snapshots.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+#
+# Publish the latest development release (a.k.a. daily snapshot)
+#
+
+. ~/.server.config || exit $?
+
+# auto-detect the latest 'major' version number
+SQUID_RELEASE=`ls -1 $SQUID_VCS_PATH | cut -d- -f2 | sort -h | tail -n 1`
+
+# Hostname of CI build server
+SQUID_CI_SERVER=buildmaster.squid-cache.org
```

Comment:
As we have recently discussed elsewhere, "CI" is too general to mean something here, especially in context where nearly all servers participate in CI, to one extent or another. Here, we know how this variable is used so we can (and, hence, should) be more specific:

```suggestion
# Hostname of the server where snapshots are built
SQUID_SNAPSHOT_BUILDER_SERVER=buildmaster.squid-cache.org
```

or, simply,

```suggestion
# Hostname of the server where snapshots are built
SQUID_BUILD_SERVER=buildmaster.squid-cache.org
```

or, if we want to reuse the "BuildFarm" terminology used in the error messages below:

```suggestion
# Hostname of the server where snapshots are built
SQUID_BUILDFARM_SERVER=buildmaster.squid-cache.org
```

As usual, I do not insist on any specific wording.

Reply:
Aye. I put it back in here with the earlier name so we can try to clarify the naming here.

My counterpoints on these suggestions are:

 * "BUILDFARM" is an artifact of the current practice. It is not a constant property of this server - this hostname has at various times been master, east, west, and one of Treehouse Networks machines. It might eventually become github.

 * "CI" - is the literal definition of what this server is doing during the release process. Literally assembling/integrating the source code/content before publication.

Reply:
Looks like `SQUID_SNAPSHOT_BUILDER_SERVER` or `SQUID_BUILD_SERVER` are the current winners. Please pick one and move on.

> "CI" - is the literal definition of what this server is doing during the release process.

I can only repeat that "CI" is too general -- it obviously matches what the server is doing, but it also matches what all other CI servers are (and will be) doing. It is like calling an animal in a room full of cats a "cat". We can do better, and any of the imperfect change request suggestions are better than "CI". Pick the one you dislike the least and move on (or suggest something even better).